build.gradle

```diff
@@ -14,8 +14,8 @@ group = project.maven_group
 repositories {
     mavenCentral()
     maven {
-        name = "gaucho-matrero-repo"
-        url = "https://gaucho-matrero.github.io/maven/"
+        name = "MarvionKiritoRepo"
+        url = "https://github.com/MarvionKirito/maven"
```

Comment:
Let's keep the maven repo on gauchomatrero. If you were to ever leave the project, we would be ******

Reply:
Once we get this working, I will merge this and it will become main